packages/errors/tsconfig-lint.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["./src/**/*", "./test/**/*"],
+  "include": ["./src/**/*", "./scripts/**/*", "./test/**/*"],
```

Comment:
This kind of thing meant there were lint errors at the top of those files. Might also have been the real problem behind what I saw on windows in CI. Had to include a few folders here and there to fix it.